package.json

```diff
@@ -386,7 +386,9 @@
     "test:lint:fix": "eslint src tests --ext ts --fix",
     "test:unit": "vitest -c ./vitest.conf.ts",
     "test:e2e": "wdio run ./tests/e2e/wdio.conf.ts",
-    "watch": "npm run build:dev -- --watch"
+    "watch": "npm run build:dev -- --watch",
+    "vscode:prepublish": "npm run prepare-binary",
```

Comment:
We might want to run this script on `postinstall` as well as we move from wasm. Maybe not in this PR, but something to keep note of.

Note also that since there is no host process identification in the current scripts, we'll need a third script later on to identify the host's platform and then download the proper binary.